SYCL/cgeist/single_task.cpp

```diff
@@ -0,0 +1,33 @@
+// RUN: %clangxx -fsycl -fsycl-targets=spir64-unknown-unknown-syclmlir %s -o %t.out 2>&1
```

Comment:
Tests usually set `-fsycl-targets` to the substitution variable `%sycl_triple`. 

If we want to limit this to SPIR-V based targets for now, because only that is supported by the MLIR-based compilation flow, we should probably add an `UNSUPPORTED` clause for the other backends, e.g. `UNSUPPORTED: cuda || hip`, similar to how other tests do that.

Reply:
We require `-syclmlir` to be the suffix in order to invoke `cgeist`. I tried `%sycl_triple-syclmlir`, which doesn't work.
Do you think `%sycl_triple-unknown-unknown-syclmlir` makes sense?

Reply:
Maybe the `llvm-lit` substitution mechanism doesn't work correctly in that case, because it doesn't match `%sycl_triple-syclmlir` against the substitution `%sycl_triple`, leaving it unsubstituted. 

We could simply attach `-syclmlir` to the `sycl_triple` in `lit.cfg.py` for our branch if the substitution doesn't work otherwise.